src/mower_logic/src/mower_logic/behaviors/UndockingBehavior.cpp

```diff
@@ -37,7 +37,7 @@ std::string UndockingBehavior::state_name() {
 }
 
 Behavior *UndockingBehavior::execute() {
-  static double next_undock_angle = 0.0;
+  static bool seedRqd = true;
```

Comment:
The name should use under_scores instead of camelCase like the remaining code and "rqd" is unclear. Maybe `random_seeded = true` and then reverse the conditions?

Reply:
I guess that means "required", so how about `rng_seeding required`?